java/src/main/java/ai/onnxruntime/OrtSession.java

```diff
@@ -808,14 +808,44 @@ public void addTensorrt(int deviceNum) throws OrtException {
       addTensorrt(OnnxRuntime.ortApiHandle, nativeHandle, deviceNum);
     }
 
+    /**
+     * The NNAPI option to use. Needs to be kept in sync with the NNAPIFlags enum in
+     * include/onnxruntime/core/providers/nnapi/nnapi_provider_factory.h
+     */
+    public enum NNAPIFlags {
```

Comment:
I am not sure if we should put the enum here, or should it be moved to where other enums are, such as `public enum OptLevel`